kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+
+  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" >
```

Comment:
@radinamatic The code within this PR addresses that problem by having
breadcrumb --> channel switch --> search 
you can find it in the `toolbar index.vue`

<img width="654" alt="screen shot 2016-08-15 at 8 56 52 am" src="https://cloud.githubusercontent.com/assets/12800136/17670348/589da2b0-62c6-11e6-9d47-85413587e0ed.png">